cmd/crictl/display.go

```diff
@@ -24,22 +24,23 @@ import (
 )
 
 const (
-	columnContainer = "CONTAINER"
-	columnImage     = "IMAGE"
-	columnImageID   = "IMAGE ID"
-	columnCreated   = "CREATED"
-	columnState     = "STATE"
-	columnName      = "NAME"
-	columnAttempt   = "ATTEMPT"
-	columnPodID     = "POD ID"
-	columnNamespace = "NAMESPACE"
-	columnSize      = "SIZE"
-	columnTag       = "TAG"
-	columnDigest    = "DIGEST"
-	columnMemory    = "MEM"
-	columnInodes    = "INODES"
-	columnDisk      = "DISK"
-	columnCPU       = "CPU %"
+	columnContainer  = "CONTAINER"
+	columnImage      = "IMAGE"
+	columnImageID    = "IMAGE ID"
+	columnCreated    = "CREATED"
+	columnState      = "STATE"
+	columnName       = "NAME"
+	columnAttempt    = "ATTEMPT"
+	columnPodID      = "POD ID"
+	columnPodRuntime = "Runtime"
```

Comment:
```suggestion
	columnPodRuntime = "RUNTIME"
```

Reply:
Updated with the changes: Show `Runtime` when `crictl pods -v`.

```diff
+       columnPodRuntime = "RUNTIME"

+               fmt.Printf("%s: %s\n",
+                       strings.Title(strings.ToLower(columnPodRuntime)),
+                       getSandboxesRuntimeHandler(pod))
+
```

PTAL. Thanks!